core-bundle/tests/EventListener/DataContainer/DisableAppConfiguredSettingsListenerTest.php

```diff
@@ -26,16 +26,12 @@ public function testAnnotatedCallbacks(): void
         $listener = $this->createListener();
 
         $annotationReader = new AnnotationReader();
+
+        /** @var callback $annotation */
```

Comment:
Just in case you're wondering as well: I just noticed that the CS fixer's `PhpCsFixer\Fixer\Phpdoc\PhpdocTypesFixer` transforms `Callback` to `callback` :man_facepalming:. It doesn't really matter though, because the annotation still works.

Reply:
I have adjusted this in contao/easy-coding-standard@251e23cf9435e7e9dfd867f5606024df5424ca32.